php/libraries/NDB_Client.class.inc

```diff
@@ -104,7 +112,8 @@ class NDB_Client
             . "default-src 'self' 'unsafe-inline'; "
             . "script-src 'self' 'unsafe-inline' 'unsafe-eval'; "
             . "font-src 'self' data:; "
-            . "img-src 'self' data:"
+            . "img-src 'self' data:; "
+            . $config_additions
```

Comment:
drawbacks are that the user need to know to add a semicolon after each addition and they are added as a signle string.

ideally one would make this a multi-row field and each addition goes on its own row (the code then concatenates them and adds semi-colons)